.vscode/tasks.json

```diff
@@ -31,7 +31,7 @@
         },
         {
             "label": "run k6test",
-            "command": "k6 run --out influxdb=http://172.17.0.1:8086/reportdb k6-tests/challenge${input:testToRun}test.js",
+            "command": "/workspace/scripts/k6 run --out influxdb=http://172.17.0.1:8086/reportdb k6-tests/challenge${input:testToRun}test.js",
```

Comment:
I just ran my go-lang and xk6 build chain from the scripts folder so a special version of k6 ends up there... This will break all other tests etc... So need to split it out to one "extended k6" buildstep that we do later in the workshop.